extensions/3DTILES_implicit_tiling/0.0.0/schema/subtree.schema.json

```diff
@@ -2,37 +2,39 @@
   "$schema": "http://json-schema.org/draft-04/schema",
   "title": "Subtree",
   "type": "object",
-  "description": "An object describing the availability of tiles, children, metadata, layers, and content in a subtree.",
+  "description": "An object describing the availability of tiles and content in a subtree, as well as availability of children subtrees.",
   "properties": {
     "buffers":{
       "type": "array",
-      "description": "An array of buffers storing binary availability metadata.",
+      "description": "An array of buffers.",
       "items": {
         "$ref": "buffer.schema.json"
       }
     },
     "bufferViews":{
       "type": "array",
-      "description": "An array of bufferViews indicating whether each element is available. Each buffer is a bitstream. If an element is available, its bit is 1, and if it is unavailable, the bit is 0. An element can correspond to a tile within a subtree, or a child tile of the subtree.",
+      "description": "An array of bufferViews.",
       "items": {
         "$ref": "bufferView.schema.json"
       }
     },
     "tileAvailability": {
-      "description": "The availability of tiles in the subtree. If not all available or unavailable, the size of the buffer will be (N ^ (subtreeDepth + 1) - 1) / (N - 1) bits where N is 4 for tiling scheme `QUADTREE` and 8 for `OCTREE`.",
+      "description": "The availability of tiles in the subtree. The availability bitstream is a 1D boolean array where tiles are ordered by their level in the subtree and Morton index within that level. A tile's availability is determined by a single bit, 1 meaning a tile exists at that spatial index, and 0 meaning it does not. The number of elements in the array is `(N ^ (subtreeDepth + 1) - 1) / (N - 1)` where N is 4 for tiling scheme `QUADTREE` and 8 for `OCTREE`. Availability may be stored in a bufferView or as a constant value that applies to all tiles. If a non-root tile's availability is 1 its parent tile's availability must also be 1.",      
```

Comment:
I fleshed out the description and included some availability rules that had been included in the schema in https://github.com/CesiumGS/3d-tiles/tree/implicit-tiling-3.